content/learn/book/getting-started/ecs/_index.md

```diff
@@ -174,4 +174,4 @@ fn main() {
 
 Note that we have used `.chain()` on the two systems. This is because we want them two to run in exactly the order they're listed in the code: with `update_people` occurring before `greet_people`.
 If they weren’t, the name might change after we greet the people.
-But we don’t add the `hello_world` system to this chain, because it doesn’t matter when it runs. That way Bebmvy can run `hello_world` while the other systems are running.
+But we don’t add the `hello_world` system to this chain, because it doesn’t matter when it runs. This way, bevy can run `hello_world` while the other systems are running.
```

Comment:
```suggestion
But we don’t add the `hello_world` system to this chain, because it doesn’t matter when it runs. This way, Bevy can run `hello_world` while the other systems are running.
```